tests/themes/run.sh

```diff
@@ -12,6 +12,17 @@ if ! [ -f CMakeLists.txt ]; then
 fi
 source_dir="$PWD"
 
+# Either the build dir is passed in $CMAKE_BINARY_DIR or we guess based on $PWD
+# Same as in tests/run.sh.
+build_dir="$CMAKE_BINARY_DIR"
+if [ -z "$build_dir" ]; then
+    if [ -d "$source_dir/build" ]; then
+        build_dir="$source_dir/build"
+    else
+        build_dir="$source_dir"
+    fi
+fi
```

Comment:
Didn't I already see this commit in some other PR...?
Anyway, I'm not really sure what to review here. `tests/run.sh` is @blueyed's playground. ;-) If it helps: 👍